recipe/meta.yaml

```
@@ -10,16 +10,18 @@ source:
   sha256: 9775fd76d8a743d39b87df38af1cd04a58c9b5a5242d5a6350ef343d06814ab6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<3.0]
```

```suggestion
  skip: true  # [py<36]
```
Skipping py<36 will be better as versions 3.0-3.5 are not used anymore